os/board/artik05x/src/artik05x_symtab.c

```diff
@@ -0,0 +1,104 @@
+/*****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <stdint.h>
+#include <stdio.h>
+#include <assert.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <arch/board/symtab.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define MAX_SYMBOL_NAME_LEN 256
+/****************************************************************************
+ * Private Variables
+ ****************************************************************************/
+static uint32_t symtab_base_addr = 0;
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+int up_symtab_flashdrv_init(void)
+{
+	char *a;
+	char *b;
+	int index = 0;
+	int index_1 = 0;
+	char s[MAX_SYMBOL_NAME_LEN];
+
+	a = CONFIG_ARTIK05X_FLASH_PART_NAME;
+	b = CONFIG_ARTIK05X_FLASH_PART_LIST;
+
```

Comment:
It will be good to perform null check or pre processor check for 'a' and 'b' and throw an error if these config values are not defined.

Reply:
Yes, config checks have to be done.